test/integration/targets/setup_cron/tasks/main.yml

```diff
@@ -19,17 +19,23 @@
 
 - when: faketime_pkg | default(false, true)
   block:
-    - name: install cron and faketime packages
+    - name: install faketime packages
       package:
         name: '{{ faketime_pkg }}'
       register: faketime_package_installed
       until: faketime_package_installed is success
+      when: ansible_distribution != 'Alpine'
+
+    - name: install faketime packages - Alpine
+      command: apk add -U {{ faketime_pkg }} --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
```

Comment:
Any reason this uses `command` rather than the `apk` module?

Reply:
Nevermind, I see the explanation in a later comment.

Reply:
The same reason I commented on at https://github.com/ansible/ansible/pull/70946#discussion_r461680298

`apk` isn't core, and I see no reason to vendor it for these simple use cases in tests.